var/spack/repos/builtin/packages/python/package.py

```diff
@@ -114,15 +119,17 @@ def install(self, spec, prefix):
                       'user configurations are present.').format(self.version))
 
         # Need this to allow python build to find the Python installation.
-        env['PYTHONHOME'], env['PYTHONPATH'] = prefix, prefix
-        env['MACOSX_DEPLOYMENT_TARGET'] = '10.6'
+        spack_env.set('PYTHONHOME', prefix)
+        spack_env.set('PYTHONPATH', prefix)
+        spack_env.set('MACOSX_DEPLOYMENT_TARGET', '10.6')
```

Comment:
@mathstuf It looks like you added this in #188. Can you comment on this? Was it added because RPATH support didn't work otherwise in 10.5 and earlier?